main_test.go

```diff
@@ -20,42 +21,60 @@ import (
 const DNS_DOMAIN = "cluster.local"
 
 var (
-
 	config    *rest.Config
 	testenv   env.Environment
-	namespace string
 	model     *matrix.Model
 	cs        *kubernetes.Clientset
 	ma        *matrix.KubeManager
 	ctx       = context.Background()
 	logger    *zap.Logger
 )
 
+
 func init() {
 	var err error
-	if logger, err = zap.NewProduction(); err != nil {
+	if logger = NewLoggerConfig(); err != nil {
 		log.Fatal(err)
```

Comment:
looks like no need to check err here, as no err is returned.

Reply:
we should fix this logger thing. Suggestion of another library, or improvement in the setup, the worst thing is the need to pass the logger down the stack manager/models etc